pkg/controllerhelpers/scylla.go

```diff
@@ -218,3 +219,19 @@ func EnsureNodeConfigCondition(status *scyllav1alpha1.NodeConfigStatus, cond *sc
 
 	*existingCond = *cond
 }
+
+func IsScyllaClusterRolledOut(sc *scyllav1.ScyllaCluster) bool {
```

Comment:
nit: this would need to be consolidated to avoid duplication in our e2e suite

Reply:
I assume this functionality shall be moved from e2e to here then.

Reply:
I don't think this is gonna be used but theoretically the e2e would be using this one and adding extra logging